src/tests/system/requirements.txt

```diff
@@ -5,4 +5,5 @@ git+https://github.com/next-actions/pytest-mh
 git+https://github.com/next-actions/pytest-ticket
 git+https://github.com/next-actions/pytest-tier
 git+https://github.com/next-actions/pytest-output
-git+https://github.com/SSSD/sssd-test-framework
+#git+https://github.com/SSSD/sssd-test-framework
```

Comment:
This part looks really weird, Could you explain why we move away from /sssd-test-framework upstream?

Reply:
okay, tests uses the code from sssd-test-framework which is not merged yes to run the test,
here is PR from sssd-test-framework, https://github.com/SSSD/sssd-test-framework/pull/112

Once that PR merged will update it.

Reply:
Can you mark it as `blocked` or convert it to draft until it is ready for review?

Reply:
okay.

Reply:
Framework PR is merged, removing the `blocked` label.